docs/android-core-new/room-metadata.mdx

```diff
@@ -1,64 +1,48 @@
 ---
 title: Room Metadata
-description: All metadata pertaining to a meeting.
-sidebar_position: 11
+description: >-
+  Learn about managing room metadata in your Android application using Dyte
+  Docs.
+sidebar_position: 1
 tags:
-  - web-core
+  - mobile-core
   - room-metadata
 ---
 
 # Room Metadata
 
 All metadata pertaining to a meeting is stored in `meeting.meta`. This includes:
 
-- `viewType`: Indicates the type of the meeting, possible values are `WEBINAR`,
-  `GROUP_CALL`.
-- `roomType`: Indicates the meeting is a group-call or a webinar.
+- `roomName`: The name of the room the current participant is connected to.
+- `meetingId`: The unique identifier of the meeting.
+- `meetingType`: Indicates the meeting is a group-call or a webinar.
```

Comment:
(suggestion): This works for now, but we can make it same as web so that it tells about `DyteMeetingType` and its possible values.